Sources/CAudioKit/CoreAudio/DSPBase.mm

```diff
@@ -1,14 +1,22 @@
 // Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
 
-#import "DSPBase.h"
-#import "ParameterRamper.h"
-#import <map>
-#import <string>
-
+#if __APPLE__
+#include "DSPBase.h"
+#include "ParameterRamper.h"
+#else // __APPLE__
+#include "DSPBase.h"
+#include "ParameterRamper.h"
+#endif // __APPLE__
```

Comment:
I don't understand how the if and else are different?

Reply:
Fixed!